flight/targets/brainre1/board-info/board_hw_defs.c

```diff
@@ -68,7 +71,12 @@ static const struct pios_annunc pios_annuncs[] = {
 		},
 		.remap = 0,
 		.active_high = true,
-	}
+	},
+#if defined(PIOS_INCLUDE_RE1_FPGA)
```

Comment:
It is linked in the bootloader. Without it, there wouldn't be any LEDs on RE1 when the BL is active ;)